tools/gepa/generate_templates.py

```
@@ -0,0 +1,313 @@
+    out_dir = args.out_dir or (root / "templates" / "generated")
+
+    # Discover and build specs
+    specs = discover_templates(root)
+    if not specs:
+        eprint("No templates found.")
+        return 1
+
+    # Configure LM and run optimization
+    try:
+        configure_lm(args)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)